Solutions/NXLogDnsLogs/Data Connectors/NXLogDnsLogs.json

```diff
@@ -0,0 +1,102 @@
+{
+    "id": "NXLogDnsLogs",
+    "title": "NXLog DNS Logs",
+    "publisher": "NXLog",
+    "descriptionMarkdown": "The NXLog DNS Logs data connector uses Event Tracing for Windows ([ETW](https://docs.microsoft.com/windows/apps/trace-processing/overview)) for collecting both Audit and Analytical DNS Server events. The [NXLog *im_etw* module](https://nxlog.co/documentation/nxlog-user-guide/im_etw.html) reads event tracing data directly for maximum efficiency, without the need to capture the event trace into an .etl file. This REST API connector can forward DNS Server events to Azure Sentinel in real time.",
+    "additionalRequirementBanner": "This data connector depends on parsers based on Kusto functions deployed with the Azure Sentinel Solution to work as expected. The [**NXLog_parsed_DNS_Server_ASim_view**](https://aka.ms/sentinel-nxlogdnslogs-parser) is designed to leverage Azure Sentinel's built-in DNS-related analytics capabilities.",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "NXLog_parsed_DNS_Server_ASim_view",
+            "baseQuery": "NXLog_parsed_DNS_Server_ASim_view"
+        }
+    ],
+    "sampleQueries": [
+        {
+            "description" : "DNS Server top 5 hostlookups",
+            "query": "NXLog_parsed_DNS_Server_ASim_view\n| summarize count() by Domain\n| take 5\n| render piechart title='Top 5 host lookups'"

```

Comment:
Please use ASimDnsMicrosoftNXLog as the table/view name in all sample queries.

Reply:
Applied changes requested in commit [1c03252](https://github.com/Azure/Azure-Sentinel/pull/3056/commits/1c032529ea6917003a6cbe97ed125729803160c1).